changelog.md

```diff
@@ -3,21 +3,49 @@
 ## 2.2.0~rc1
 
 ### Web applications
-* Improved accessibility of Journalist Interface via semantic markup (#6165)
-* Improve handling of HOTP tokens setup (#5599)
-* Disabled caching of passphrases in gpg-agent (#
-* Added additional defensive HTTP headers to Apache config (#6187)
-* Updated dependencies: click 6.7 to 8.0.3; flask 1.0.2 to 2.0.2; flask-wtf 0.14.2 to 1.0.0; jinja2 2.11.3 to 3.0.2; markupsafe 1.1.1 to 2.0.1; redis 3.3.6 to 3.5.3; rq 1.1.0 to 1.10.0; werkzeug 0.16.0 to 2.0.2 (#6190, #6217)
+
+* Refactor and simplify code for creating a new source (#6087).
+* Timestamps will now be displayed using the language's "long" format (#6144).
+* The "Forgot your codename?" hint will only be shown on the initial login (#6130).
+* Disable caching of GPG passphrases to ensure the correct passphrase must be passed every time (#6174).
+* Set minimum length requirements on HOTP and TOTP secrets (##6191).
+* Improve security-related/defensive HTTP headers used in the source and journalist interfaces (#6187).
+* The "Refresh codename" feature was removed because of potential user confusion (#6195).
+* Refactor code related to encryption (#6160).
+* Stop explicitly flushing the database session (#6223).
+* Upgrade to Flask 2.0 and associated refactoring (#6217).
+* Improve accessibility of journalist interface by using semantic HTML5/ARIA markup (#6165).
+* Information corresponding to deleted journalists will now be associated with a reserved "deleted" account internally. (#6225).
 
 ### Operations
 
-* Migrated kernel version from 5.4.136 to 5.15.18, for newer hardware support (#6242)
-* Added support for custom hostnames to `./securedrop-admin verify` (#6154)
-* Removed references to v2 onion config (#6169)
+* Update Tor to 0.4.6.9 (#6227).
```

Comment:
I'm kindof on the fence about including Tor updates in the SD changelog anymore, IMO we should be decoupling them from the release cycle and striving just to stay on latest stable. We updated Tor a week ago so it's not really a 2.2.0 thing.

Reply:
Makes sense, removed.